Tests/TLSEvaluationTests.swift

```diff
@@ -414,7 +422,15 @@ class TLSEvaluationExpiredLeafCertificateTestCase: BaseTestCase {
         waitForExpectations(timeout: timeout, handler: nil)
 
         // Then
+    #if os(iOS) || os(macOS)
+        if #available(iOS 10.1, macOS 10.12.0, *) {
+            XCTAssertNotNil(error, "error should not be nil")
+        } else {
+            XCTAssertNil(error, "error should be nil")
+        }
+    #else
         XCTAssertNil(error, "error should be nil")
+    #endif
     }
```

Comment:
I also fixed the `macOS` check, as it was just `(macOS)` before, which doesn't work but for some reason compiles.